README.md

```diff
@@ -7,7 +7,7 @@
 Describe the differences between a SQL and NoSQL database, and when you might use each.
 
 ```text
-Your answer...
+SQL databases are relational, so items on different tables can be linked by a join table. NoSQL are document based, so they are more flexible about how data is entered, but not designed with the rigid structure of tables to make the data conform to a specified structure as well.
 ```
```

Comment:
:+1: although not all NoSQL databases are document-based: https://www.3pillarglobal.com/insights/exploring-the-different-types-of-nosql-databases